.github/workflows/tests.yml

```diff
@@ -54,34 +54,14 @@ jobs:
           disableRateLimiting: true
           public_baseurl: "http://10.0.2.2:8080/"
 
-      - name: AVD cache
-        uses: actions/cache@v3
-        id: avd-cache
-        with:
-          path: |
-            ~/.android/avd/*
-            ~/.android/adb*
-          key: avd-${{ matrix.api-level }}
-
-      - name: create AVD and generate snapshot for caching
```

Comment:
caching the AVD is causing strange problems for the `playstore` emulator image where the expected AVD path changes between runs 

```
PANIC: Broken AVD system path. Check your ANDROID_SDK_ROOT value [/usr/local/lib/android/sdk]!
```

Creating a new AVD image each run fixes the problem but does mean the initial start time becomes slower